### PR TITLE
Tool disk syntax change

### LINKED ISSUE #2006: projects/kubernetes run boot script fails
I want to run `boot-master.sh` to boot a master, but I get an error as follow:
```sh
$ ./boot-master.sh
+ rm -f kube-master-disk.img
+ ../../bin/linuxkit run -cpus 2 -mem 4096 -disk-size 4096 kube-master
flag provided but not defined: -disk-size
USAGE: linuxkit run hyperkit [options] prefix

'prefix' specifies the path to the VM image.

Options:
  -cpus int
    	Number of CPUs (default 1)
  -data string
    	Metadata to pass to VM (either a path to a file or a string)
  -disk value
    	Disk config. [file=]path[,size=1G]
  -hyperkit string
    	Path to hyperkit binary (if not in default location)
  -ip string
    	IP address for the VM
  -mem int
    	Amount of memory in MB (default 1024)
  -networking string
    	Networking mode. Valid options are 'docker-for-mac', 'vpnkit[,socket-path]' and 'none'. 'docker-for-mac' connects to the network used by Docker for Mac. 'vpnkit' connects to the VPNKit socket specified. If socket-path is omitted a new VPNKit instance will be started and 'vpnkit_eth.sock' will be created in the state directory. 'none' disables networking.` (default "docker-for-mac")
  -state string
    	Path to directory to keep VM state in
  -vsock-ports string
    	List of vsock ports to forward from the guest on startup (comma separated). A unix domain socket for each port will be created in the state directory
```

This error looks like being linuxkit flag is changed.

### PR DESCRIPTION
**Pull request by riyazdf**
- change `-disk-size` to `-disk <optional_path>,size=$SIZE`
- bonus: update `moby run` to `linuxkit run`

Closes #2006 

</img>

### CODE REVIEW

**Comment by thebsdbox** [comment]
I agree with a flag reduction, I'd like a quick discussion to agree on a standard for the "main" flags.

File: pkg/swap/README.md [hunk 1]
```diff
@@ -79,5 +79,5 @@ The sample command to run the enclosed is:
 
 ```
 moby build swap.yml
-moby run -disk-size 4096 swap
+linuxkit run -disk size=4096 swap
```
Comment (justincormack) [inline pkg/swap/README.md]: can you put the `M` or `G` in as we may change the default...

File: projects/kubernetes/boot-master.sh [hunk 2]
```diff
@@ -2,4 +2,4 @@
 disk="kube-master-disk.img"
 set -x
 rm -f "${disk}"
-../../bin/linuxkit run -cpus 2 -mem 4096 -disk-size 4096 kube-master
+../../bin/linuxkit run -cpus 2 -mem 4096 -disk "${disk}",size=4096 kube-master
```
Comment (justincormack) [inline projects/kubernetes/boot-master.sh]: and here `4096M` or `4G`

File: projects/kubernetes/boot-node.sh [hunk 3]
```diff
@@ -5,4 +5,4 @@ shift
 disk="kube-${name}-disk.img"
 set -x
 rm -f "${disk}"
-../../bin/linuxkit run -cpus 2 -mem 4096 -disk-size 4096 -disk "${disk}" -data "${*}" kube-node
+../../bin/linuxkit run -cpus 2 -mem 4096 -disk "${disk}",size=4096 -data "${*}" kube-node
```
Comment (justincormack) [inline projects/kubernetes/boot-node.sh]: ditto

File: pkg/swap/README.md (hunk 1 above)
Comment (riyazdf) [inline pkg/swap/README.md]: changed to `4G` in all the places

**Comment by riyazdf** [comment]
@thebsdbox: mind opening a separate issue for `linuxkit` flags?  This CLI change was already merged earlier so this PR is just to get things working with it properly

**Pull request merged**